mesonbuild/linkers/detect.py

```diff
@@ -112,7 +112,7 @@ def guess_nix_linker(env: 'Environment', compiler: T.List[str], comp_class: T.Ty
                      extra_args: T.Optional[T.List[str]] = None) -> 'DynamicLinker':
     """Helper for guessing what linker to use on Unix-Like OSes.
 
-    :compiler: Invocation to use to get linker
+    :compiler: Invocation to use to gThe issue from the last one appears to be et linker
```

Comment:
Typo.

Reply:
Gah, my new trackball's scroll wheel is way to sensitive to clicks and I keep having this happen.